### ISSUE TITLE
[a11y]: Color contrast does not meet WCAG standards

### ISSUE THREAD

**Issue by CBID2** [narashimha05/GitResume#21]
### Description

I did an accessibility check and found that the foreground color for the "Get Started" button does not met WCAG expectations, making it inaccessible for users with low-vision. 

### Screenshot

![Image](https://github.com/user-attachments/assets/5ddd9e0b-3062-4021-a120-9fbe95731142)

### Proposed Solution
I proposed to use a darker green for the button as shown in the image below. This would keep the original color scheme and make the button more accessible.  
![Image](https://github.com/user-attachments/assets/ba00e4d3-4b24-4b0a-b3fa-763a4b6ca7bf)

**Comment by narashimha05** [comment]
yeah please go ahead and would love to see your contribution !
